arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Option.kt

```diff
@@ -904,12 +904,17 @@ public fun <A> A.some(): Option<A> = Some(this)
 
 public fun <A> none(): Option<A> = None
 
+@Deprecated("use fold instead", ReplaceWith("fold(MA)", "arrow.core.fold"))
```

Comment:
This fold doesn't exist, should we just deprecate to `fold(Monoid.option(MA))` ?